sqlmesh/dbt/basemodel.py

```diff
@@ -170,17 +170,29 @@ def config_name(self) -> str:
         return f"{self.package_name}.{self.name}"
 
     @property
-    def sql_name(self) -> str:
+    def full_sql_name(self) -> str:
         """
-        Get the sqlmesh model name
+        Get the fully qualified model name
 
         Returns:
-            The sqlmesh model name
+            The fully qualified model name
         """
         return ".".join(
             part for part in (self.database, self.table_schema, self.table_name) if part
         )
 
+    def sql_name(self, context: DbtContext) -> str:
+        """
+        Get the sqlmesh model name
+
+        Returns:
+            The sqlmesh model name
+        """
+        # TODO add back in conditional database
+        # database = self.database if self.database != context.default_database else None
+        database = self.database
+        return ".".join(part for part in (database, self.table_schema, self.table_name) if part)
```

Comment:
Why is this equivalent to `full_sql_name`? Should we just call that with TODO?

Reply:
Once the renderer supports translating fqn or model name to physical table, the intention is to use the commented out logic. @eakmanrq is working on adding the renderer logic.